api/envoy/api/v2/core/grpc_service.proto

```diff
@@ -86,6 +86,27 @@ message GrpcService {
         }
       }
 
+      // [#next-free-field: 10]
+      message StsService {
```

Comment:
What's the situation with documentation?

Reply:
The comment ```// [#next-free-field: 10]``` was added by command ```./tools/proto_format.sh fix```, I removed it and added ```--no-verify``` to push the commit.

Reply:
@JimmyCYJ next-free-id is required, also you should add note how this message is used.

Reply:
@lizan I have updated comments to the message. Please take a look. Thanks!